### PR TITLE
chore(space-wizard) : improved formatting and linting

### PR DESCRIPTION
**Pull request by mitchharpur**
improved formatting and linting for space wizard. nothing else changed here

### CODE REVIEW

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: 07411af5bee861bab7bdd74a8b5912ca8f046aa5
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**
  * Your subject line is longer than 50 characters

Guidelines are available at https://github.com/fabric8io/fabric8-ui

---
This message was auto-generated by https://gitcop.com

File: src/app/space-wizard/components/wizard-dynamic-step/wizard-dynamic-step.component.ts [hunk 1]
```diff
@@ -1,66 +1,72 @@
...
 @Component({
-  host: {
-    'class': 'wizard-step'
-  },
-  selector: 'wizard-dynamic-step',
-  templateUrl: './wizard-dynamic-step.component.html',
-  styleUrls: ['./wizard-dynamic-step.component.scss'],
-  providers: [IAppGeneratorServiceProvider.FactoryProvider]
-})
+             host       : {
```
Comment (joshuawilson) [inline src/app/space-wizard/components/wizard-dynamic-step/wizard-dynamic-step.component.ts]: this indent doesn't look right, if it is in the linter then we need to update the linter.

File: src/app/space-wizard/components/wizard-dynamic-step/wizard-dynamic-step.component.ts [hunk 2]
```diff
@@ -1,66 +1,72 @@
...
   multiSelectSettings: IMultiSelectSettings = {
-    pullRight: false,
-    enableSearch: true,
-    checkedStyle: 'checkboxes',
-    buttonClasses: 'btn btn-default',
-    selectionLimit: 0,
-    closeOnSelect: false,
-    showCheckAll: true,
-    showUncheckAll: true,
+    pullRight           : false,
```
Comment (joshuawilson) [inline src/app/space-wizard/components/wizard-dynamic-step/wizard-dynamic-step.component.ts]: Is this in the linter? we don't do this anywhere else

**Comment by mitchharpur** [comment]
i will submit a formatting pull request another time

**Pull request closed without merging**